Reference/Security/Security-hardening/index.md

```diff
@@ -0,0 +1,59 @@
+# Umbraco Security Hardening
+
+Here you find some tips and trick for hardening the security of your Umbraco installation.
+
+## Rename your Umbraco-folder
+
+By default the login page of Umbraco is available at the path /umbraco/. This page is the entrance to your installation and it’s considered a good practice to rename your path to a more secure path.
```

Comment:
Are there any packages that rely on the umbraco folder being called 'umbraco' - I know in the past I've accidentally created some.

Reply:
Not that I'm aware of, but we will found it out pretty quickly indeed. Do you know what the correct way is in Umbraco to determine the 'Umbraco-path'. And maybe we can give that as hint to package developers to use.

Reply:
There's lots of packages with hard coded "/umbraco" paths ;) they shouldn't be hard coded but they are.

Reply:
Hi @Shazwazza, and what should we tell these package developers? Which code should they use to get the correct path? I don't think that we should read it from the web.config, or do we? And ../../ is a possibility neither I guess?

If you can tell me how to do it, I can document that as well. Thanks!